### PR TITLE
VTCCA-4164-update-template-di

### PR DESCRIPTION
**Pull request by zahiriqbal**

### CODE REVIEW

File: test/controllers/manageDetails/UpdateOrganisationDetailsSpec.scala [hunk 1]
```diff
@@ -247,10 +247,14 @@ class UpdateOrganisationDetailsSpec extends VoaPropertyLinkingSpec {
...
-      updateBusinessPhoneView = new views.html.details.updateBusinessPhone(mainLayout, GovukButton, GovukInput),
-      updateBusinessEmailView = new views.html.details.updateBusinessEmail(mainLayout, GovukButton, GovukInput)
+      updateBusinessAddressView =
+        new views.html.details.updateBusinessAddress(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+      updateBusinessNameView =
+        new views.html.details.updateBusinessName(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+      updateBusinessPhoneView =
+        new views.html.details.updateBusinessPhone(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+      updateBusinessEmailView =
+        new views.html.details.updateBusinessEmail(mainLayout, GovukButton, GovukInput, FormWithCSRF)
```
Comment (darrell-plant) [inline test/controllers/manageDetails/UpdateOrganisationDetailsSpec.scala:257]: how do you fancy moving these into FakeViews?

File: test/controllers/manageDetails/UpdatePersonalDetailsSpec.scala [hunk 2]
```diff
@@ -294,11 +294,11 @@ class UpdatePersonalDetailsSpec extends VoaPropertyLinkingSpec {
...
-        updateAddressView = new views.html.details.updateAddress(mainLayout, GovukButton, GovukInput),
-        updatePhoneView = new views.html.details.updatePhone(mainLayout, GovukButton, GovukInput),
-        updateMobileView = new views.html.details.updateMobile(mainLayout, GovukButton, GovukInput),
-        updateEmailView = new views.html.details.updateEmail(mainLayout, GovukButton, GovukInput),
-        updateNameView = new views.html.details.updateName(mainLayout, GovukButton, GovukInput)
+        updateAddressView = new views.html.details.updateAddress(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+        updatePhoneView = new views.html.details.updatePhone(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+        updateMobileView = new views.html.details.updateMobile(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+        updateEmailView = new views.html.details.updateEmail(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+        updateNameView = new views.html.details.updateName(mainLayout, GovukButton, GovukInput, FormWithCSRF)
```
Comment (darrell-plant) [inline test/controllers/manageDetails/UpdatePersonalDetailsSpec.scala:301]: how do you fancy moving these into FakeViews?

**Comment by darrell-plant** [comment]
Nice job! Just one more step, please...

**Comment by darrell-plant** [comment]
LGTDz 👍

File: test/controllers/manageDetails/UpdateOrganisationDetailsSpec.scala (hunk 1 above)
Comment (habeeb01) [reply test/controllers/manageDetails/UpdateOrganisationDetailsSpec.scala:257]: +1

**Comment by habeeb01** [comment]
LGTM 👍  (+1 to Darrell's comment)

**Pull request merged**